engines/freescape/gfx_opengl_shaders.cpp

```diff
@@ -140,6 +158,43 @@ void OpenGLShaderRenderer::drawTexturedRect2D(const Common::Rect &screenRect, co
 	_bitmapShader->unbind();
 }
 
+void OpenGLShaderRenderer::drawSkybox(Texture *texture, Math::Vector3d camera) {
+	OpenGLTexture *glTexture = static_cast<OpenGLTexture *>(texture);
+
+	Math::Matrix4 proj = _projectionMatrix;
+	Math::Matrix4 model = _modelViewMatrix;
+	// remove translation
+	model(3, 0) = 0.0f;
+	model(3, 1) = 0.0f;
+	model(3, 2) = 0.0f;
+
+	proj.transpose();
+	model.transpose();
+
+	Math::Matrix4 skyboxMVP = proj * model;
+	skyboxMVP.transpose();
+
+	_cubemapShader->use();
+    _cubemapShader->setUniform("mvpMatrix", skyboxMVP);
```

Comment:
I think you used the correct indentation here. It is invisible in github, but you should be able to see it in your editor (e.g. vscode)